modules/manage/partials/tiered-storage.adoc

```diff
@@ -1268,6 +1312,13 @@ To create a new topic using remote recovery and enable Tiered Storage on the new
 rpk topic create <topic_name> -c redpanda.remote.recovery=true -c redpanda.remote.write=true -c redpanda.remote.read=true
 ----
 
+=== Performing rolling restart
+
+
+=== Redirect workload to new cluster
```

Comment:
@andrwng Are there specific steps to take here or a doc we could link to?

Reply:
I don't think there are docs, and this is fairly application specific. It amounts to changing their applications to use the addresses of the new cluster

Reply:
I'm not aware of any. It's pretty application specific and amounts to changing their application code to use the addresses of the new cluster